changelogs/608-new_module-efs_tag.yml

```diff
@@ -0,0 +1,2 @@
+major_changes:
```

Comment:
new modules does not need a changelog fragment. it will be generated automatically. Please remove.

Reply:
and pipeline passes now @zeten30 🚀  
except for the changelog fragment, everything looks good to me.

Reply:
@markuman Thank you. Fragment deleted.

It was requested in one of the previous comments:)

```
Member
markuman left a comment
Thanks for your work. Looks great so far.
You need also a changelog/fragments file.
```

Reply:
yeah, my fault :)